addons/website_sale_delivery/controllers/main.py

```diff
@@ -195,7 +197,44 @@ def express_checkout_process_shipping_address(self, partial_shipping_address):
                 order_sudo.currency_id,
             ),
         } for carrier in order_sudo._get_delivery_methods()],
-        key=lambda carrier: carrier['minorAmount'])
+            key=lambda carrier: carrier['minorAmount'])
+
+    @http.route('/shop/access_point/set', type='json', auth='public', methods=['POST'], csrf=False, website=True, sitemap=False)
+    def set_access_point(self, access_point_encoded):
+        order = request.website.sale_get_order()
+        if hasattr(order.carrier_id, order.carrier_id.delivery_type + '_use_locations'):
+            use_location = getattr(order.carrier_id, order.carrier_id.delivery_type + '_use_locations')
+            access_point = use_location and (json.loads(access_point_encoded) if access_point_encoded else None) or None
+            order.write({'access_point_address': access_point})
```

Comment:
We usually write False instead of None
```suggestion
            access_point = use_location and (json.loads(access_point_encoded) if access_point_encoded else False) or False
            order.write({'access_point_address': access_point})
```